src/ring_theory/nilpotent.lean

```diff
@@ -103,6 +111,30 @@ end ring
 
 end commute
 
+section comm_semiring
+
+variable [comm_semiring R]
+
+/-- The nilradical of a commutative semiring is the nilradical -/
```

Comment:
This docstring is ~~this docstring~~ useless - did you mean to write something else?